client/components/index.js

```diff
@@ -1,33 +1,58 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { connect } from "react-redux"
 import AllCategories from './category/AllCategories'
 import SingleCategory from './category/SingleCategory'
 import CreateCostume from './costume/CreateCostume'
 import NavBar from './NavBar'
 import Home from './Home'
 import Login from './authentication/Login'
+import SingleCostume from "../components/costume/SingleCostume"
+import loadCostumesDispatch from '../redux/actions/loadCostumes';
 
 
 
-const Routes = () => {
-  return (
-    <Router>
-      <div>
-        <Route render={() => <NavBar />} />
-        <div className="container">
-        <main>
-        <Switch>
-          <Route path="/home" exact component={Home} />
-          <Route path="/categories" exact component={AllCategories} />
-          <Route path="/categories/:id" component={SingleCategory} />
-          <Route path= "costumes/add" exact component={CreateCostume} />
-          <Route path= "/login" exact component={Login} />
-        </Switch>
-        </main>
+class Routes extends Component {
```

Comment:
hi Cynthia, rather than load all costumes on the index page, could we load all categories instead, and only load all costumes for the Home component, which shows all costumes?